# Generated by Django 2.2.3 on 2019-09-27 15:01

from django.conf import settings
import django.core.validators
from django.db import migrations, models
import sNeeds.apps.chats.models


class Migration(migrations.Migration):

    dependencies = [
        ('account', '0028_auto_20190830_1536'),
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('chats', '0007_auto_20190927_0905'),
    ]

    operations = [
        migrations.AlterField(
            model_name='file',
            name='file',
            field=models.FileField(upload_to=sNeeds.apps.chats.models.get_file_upload_path),
        ),
        migrations.AlterField(
            model_name='image',
            name='image',
            field=models.ImageField(upload_to=sNeeds.apps.chats.models.get_image_upload_path, validators=[django.core.validators.FileExtensionValidator(allowed_extensions=['jpg', 'jpeg', 'png'])]),
        ),
        migrations.AlterField(
            model_name='voice',
            name='file',
            field=models.FileField(upload_to=sNeeds.apps.chats.models.get_voice_upload_path, validators=[django.core.validators.FileExtensionValidator(allowed_extensions=['mp3'])]),
        ),
        migrations.AlterUniqueTogether(
            name='chat',
            unique_together={('user', 'consultant')},
        ),
    ]
